pkg/rancher/migrations.go

```diff
@@ -132,6 +141,33 @@ func createOrUpdateConfigMap(configMapClient controllerv1.ConfigMapClient, cm *v
 	return err
 }
 
+func versionTombstone(configMapController controllerv1.ConfigMapController) error {
+	cm, err := getConfigMap(configMapController, rancherVersionTombstoneConfig)
+	if err != nil || cm == nil {
+		return err
+	}
+
+	if !semver.IsValid(rancherversion.Version) {
+		logrus.Errorf("rancher version %s is not semver compliant, skipping tombstone validation", rancherversion.Version)
+		return nil
+	}
+
+	lastVersion := cm.Data[rancherVersionKey]
+	if lastVersion != "" {
+		if !semver.IsValid(lastVersion) {
+			logrus.Errorf("Previous Rancher version %s is not semver compliant, skipping tombstone validation", lastVersion)
+			lastVersion = ""
+		}
+		if semver.Compare(lastVersion, rancherversion.Version) == 1 {
+			logrus.Fatalf("Detected Rancher downgrade from %s to %s. In order to perform a rollback of Rancher, use the Rancher Backup Restore Operator. If a suitable backup is unavailable, version tombstone validation can be temporarily disabled by deleting the rancherVersionTombstone config map in the cattle-system namespace.",
```

Comment:
It is better to use the real name of the rancherVersionTombstone config map which is `rancherversion` to not confuse users who need to look it up. 

```suggestion
			logrus.Fatalf("Detected Rancher downgrade from %s to %s. In order to perform a rollback of Rancher, use the Rancher Backup Restore Operator. If a suitable backup is unavailable, version tombstone validation can be temporarily disabled by deleting the rancherversion ConfigMap in the cattle-system namespace.",
```